apps/cli/package.json

```diff
@@ -52,7 +52,7 @@
     "puppeteer": "^13.5.0",
     "qs": "^6.9.1",
     "serve": "^11.2.0",
-    "sharp": "^0.28.3",
+    "sharp": "^0.33.0",
```

Comment:
was getting a build error working on pnpm stuff, this fixed it. sharp is the death of me.